gestalt-module/src/main/java/org/terasology/gestalt/naming/Name.java

```diff
@@ -39,62 +39,81 @@
      */
     public static final Name EMPTY = new Name("");
 
+    private final String normalised;
```

Comment:
Let's keep it as it was - less noise in the diff.

```suggestion
    private final String normalisedName;
```